css/styles.css

```diff
@@ -0,0 +1,680 @@
+* {
+  list-style: none;
+  text-decoration: none;
+}
+
+:root {
+  --black-color: #272a31;
+  --red-color: #ec5242;
+  --gray-color: #d3d3d3;
+  --nav-color: #f7f4f4;
+}
+
+a:hover {
+  cursor: pointer;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.bar {
+  display: block;
+  width: 43px;
+  height: 8px;
+  margin: 3px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+.nav-link .campain {
+  border: #94544d solid;
+}
+
+.nav-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 15px;
+}
+
+.logo img {
+  width: 10em;
+}
+
+.topbar img {
+  margin: -7px 0 4px 1px;
+}
+
+.nav-ul li {
+  display: flex;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  align-items: center;
+  text-align: center;
+  color: #272a31;
+  margin-left: 32px;
+}
+
+.cards {
+  display: flex;
+  gap: 1em;
+  justify-content: space-evenly;
+}
+
+.navbar-desktop {
+  display: flex;
+  justify-content: space-between;
+}
+
+.realbavlist {
+  display: flex;
+  gap: 3em;
+  margin: 3% 0% 0% -5%;
+}
+
+.firstpart {
+  background-image: url(../img/07.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: auto;
+}
+
+.text-container > p {
+  padding-top: 3em;
+  width: 90%;
+  margin: auto;
+  font-family: 'Lato', sans-serif;
+  font-size: x-large;
+  font-weight: 600;
+  color: var(--red-color);
+  text-align: center;
+}
+
+.creative-nexus {
+  padding-top: 1em;
+  padding-bottom: 1em;
+  width: 90%;
+  margin: auto;
+  font-size: xx-large;
+  font-weight: 900;
+  color: rgba(236, 82, 66, 0.9);
+  font-family: 'Cocogoose', sans-serif;
+}
+
+.boxed-paragraph {
+  display: block;
+  border: solid #fff;
+  border-radius: 3px;
+  padding: 1em;
+  width: 90%;
+  margin: auto;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  color: var(--black-color);
+}
```

Comment:
- [ ] Kindly make sure to use the same typography (color, font-size, fon-family...) on your whole websites paragraphs in order to meet the provided design. :wink: 

  - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/203582602-65cfe719-4b6a-4f9c-83da-d8cb96587c22.png)

  - ### PROVIDED DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/203582649-c89f7cab-924c-4a1f-8277-e78011295ae9.png)

Reply:
## thank you for noticing that part
- I increased the font weight on that part
- changed the color to the darker one that was provided in the tamplate